### ISSUE TITLE
[FEATURE] - Add Name for MSFS 2024

### ISSUE THREAD

**Issue by walkerairtrans** [invernyx/smartcars-3-bugs#398]
### Is your feature request related to a problem? Please describe.

FSUIPC7 has released a new ID number for MSFS2024 and I'd like to get the name added to the app; right now it comes across as Using 14.

![image](https://github.com/user-attachments/assets/c91711b5-b958-4294-9b7a-1d623ecbd491)

### Describe the solution you'd like

Add a name to match the ID# in the app.

Can we name it Microsoft Flight Simulator 2024 (and not name it Microsoft Flight Simulator -- so that we can tell the difference in 2020 versus 2024.)

### Describe alternatives you've considered

None

### Additional context

![smartcars3_kEkIoxBGPJ](https://github.com/user-attachments/assets/e6fbe4b9-791a-43ec-b850-7d8eb4897285)

**Comment by wimatte** [comment]
Yes, that's definitely a fix that should be easily implemented asap

**Comment by turbofandude** [comment]
Handled in 3.3+
